src/v2/Apps/_Preferences2/PreferencesApp.tsx

```diff
@@ -46,6 +51,29 @@ export const PreferencesApp: FC<PreferencesAppProps> = ({ viewer }) => {
       }
     })
 
+  const handleSubmit = async values => {
```

Comment:
I usually just inline this function so that it gets types. Otherwise you'll want to manually type it.